model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -80,16 +80,17 @@ def _build_input_alloc_and_call_args(n: Node, input_tensors_in_node_kwargs: Dict
     tensor_input_alloc = []
     op_call_args = list(n.args)
     if inputs_as_list:
-        op_call_args.pop(0)
+        _args = op_call_args.pop(0)
```

Comment:
can you add a comment explaining the two cases? why in one case you take from op_call_args and the other case you take from n.args?